chart/compass/templates/tenant-fetcher-job.yaml

```diff
@@ -0,0 +1,95 @@
+            imagePullPolicy: IfNotPresent
+            env:
+              - name: APP_DB_USER
+                valueFrom:
+                  secretKeyRef:
+                    name: compass-postgresql
+                    key: postgresql-username
+              - name: APP_DB_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    name: compass-postgresql
+                    key: postgresql-password
+              - name: APP_DB_HOST
+                valueFrom:
+                  secretKeyRef:
+                    name: compass-postgresql
+                    key: postgresql-serviceName
+              - name: APP_DB_PORT
+                valueFrom:
+                  secretKeyRef:
+                    name: compass-postgresql
+                    key: postgresql-servicePort
+              - name: APP_DB_NAME
+                valueFrom:
+                  secretKeyRef:
+                    name: compass-postgresql
+                    key: postgresql-directorDatabaseName
+              - name: APP_DB_SSL
+                valueFrom:
+                  secretKeyRef:
+                    name: compass-postgresql
+                    key: postgresql-sslMode
+              - name: APP_ENDPOINT_TENANT_CREATED
+                value: {{ .Values.global.tenantFetcher.endpoints.tenantCreated }}
+              - name: APP_ENDPOINT_TENANT_DELETED
+                value: {{ .Values.global.tenantFetcher.endpoints.tenantDeleted }}
+              - name: APP_ENDPOINT_TENANT_UPDATED
+                value: {{ .Values.global.tenantFetcher.endpoints.tenantUpdated }}
+              - name: APP_TENANT_PROVIDER
+                value: {{ .Values.global.tenantFetcher.providerName }}
+              - name: APP_CLIENT_ID
+                value: {{ .Values.global.tenantFetcher.oauth.client }}
+              - name: APP_CLIENT_SECRET
+                value: {{ .Values.global.tenantFetcher.oauth.secret }}
+              - name: APP_OAUTH_TOKEN_ENDPOINT
+                value: {{ .Values.global.tenantFetcher.oauth.tokenURL }}
+              - name: APP_MAPPING_FIELD_ID
+                value: {{ .Values.global.tenantFetcher.fieldMapping.idField }}
+              - name: APP_MAPPING_FIELD_NAME
+                value: {{ .Values.global.tenantFetcher.fieldMapping.nameField }}
+              - name: APP_MAPPING_FIELD_DISCRIMINATOR
+                value: {{ .Values.global.tenantFetcher.fieldMapping.discriminatorField }}
+              - name: APP_MAPPING_VALUE_DISCRIMINATOR
+                value: {{ .Values.global.tenantFetcher.fieldMapping.discriminatorValue }}
+            command:
+              - "/bin/sh"
+            args:
+              - "-c"
+              - "sleep 20; ./tenantfetcher; exit_code=$?; echo '# KILLING PILOT-AGENT #'; pkill -INT cloud_sql_proxy; curl -XPOST http://127.0.0.1:15020/quitquitquit; sleep 5; exit $exit_code;"
```

Comment:
There is a similar issue for that: https://github.com/kyma-incubator/compass/issues/709
Generally, all jobs that contact with DB have that sleep. In case of cronjob I think we can get of rid at all